terraform-post-packer/main.tf

```diff
@@ -64,3 +116,8 @@ module "ami_launch_permission" {
   ami_id               = each.value
   extraorg_account_ids = var.extraorg_account_ids
 }
+
+moved {
```

Comment:
We should create an issue to remove this in a future (if not the next) kraken.

Reply:
Please see #369 and commit [bd80fc7](https://github.com/cisagov/skeleton-packer/pull/355/commits/bd80fc7b435e7ddc09b009bba605fc7452a03c98).